packages/fxa-content-server/tests/functional_settings.js

```diff
@@ -3,11 +3,8 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 module.exports = [
-  'tests/functional/settings/change_email.js',
```

Comment:
If we no longer need these tests, I think we should remove the file entirely. We can always look back at git history if needed.

Reply:
We are currently using this suite (functional_settings) for our smoke tests in Stage. Since I have not been able to convert all the smoke tests into playwright tests and we don't have a playwright smoke test suite I don't want to remove the test from this suite just yet. Once all the smoke tests are converted to playwright I will delete the test from this suite too.